saleor/graphql/core/types/common.py

```diff
@@ -73,6 +73,14 @@ class ProductError(Error):
     code = ProductErrorCode(description="The error code.", required=True)
 
 
+class ProductAttributeError(ProductError):
+    attributes = graphene.List(
+        graphene.NonNull(graphene.ID),
+        description="List of attributes IDs which causes the error.",
+        required=False,
+    )
```

Comment:
I am not sure if we should create a new class or just add the `attributes` field to `ProductError`. I think it might be useful also in other mutations and checks.

Reply:
Some other errors inherit from ProductError
by adding attributes to ProductError I will add attributes to all of them

Reply:
I think this is fine.

Reply:
@maarcingebala you mean it's fine like this, or it is fine if all errors which inherit from ProductError will have `attributes`?

Reply:
Sorry, I meant the current solution is fine so having a separate ProductAttributeError that inherits from ProductError. There is no need to introduce the `attributes` field to all product errors as it doesn't make sense in some cases.